src/Exception/ExceptionInterface.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace Equip\Exception;
+
+interface ExceptionInterface
```

Comment:
This doesn't make sense to me. We shouldn't need to define a custom interface to have this functionationality.